这是第一个类别（Code Exception）的CVE所涉及的CWE，num字段表示对应该CWE的CVE的数量
``` JSON
[{
	"id": "CWE-787",
	"num": 207
},
{
	"id": "CWE-119",
	"num": 128
},
{
	"id": "CWE-264",
	"num": 59
},
{
	"id": "CWE-190",
	"num": 42
},
{
	"id": "CWE-20",
	"num": 29
},
{
	"id": "CWE-416",
	"num": 17
},
{
	"id": "CWE-189",
	"num": 15
},
{
	"id": "CWE-120",
	"num": 11
},
{
	"id": "CWE-284",
	"num": 10
},
{
	"id": "CWE-125",
	"num": 6
},
{
	"id": "CWE-122",
	"num": 6
},
{
	"id": "CWE-843",
	"num": 5
},
{
	"id": "CWE-415",
	"num": 5
},
{
	"id": "CWE-362",
	"num": 5
},
{
	"id": "CWE-908",
	"num": 4
},
{
	"id": "CWE-367",
	"num": 4
},
{
	"id": "CWE-121",
	"num": 3
},
{
	"id": "CWE-682",
	"num": 3
},
{
	"id": "CWE-755",
	"num": 3
},
{
	"id": "CWE-129",
	"num": 2
},
{
	"id": "CWE-665",
	"num": 2
},
{
	"id": "CWE-131",
	"num": 2
},
{
	"id": "CWE-476",
	"num": 2
},
{
	"id": "CWE-74",
	"num": 2
},
{
	"id": "CWE-667",
	"num": 1
},
{
	"id": "CWE-763",
	"num": 1
},
{
	"id": "CWE-502",
	"num": 1
},
{
	"id": "CWE-770",
	"num": 1
},
{
	"id": "CWE-275",
	"num": 1
},
{
	"id": "CWE-704",
	"num": 1
},
{
	"id": "CWE-252",
	"num": 1
},
{
	"id": "CWE-772",
	"num": 1
},
{
	"id": "CWE-670",
	"num": 1
},
{
	"id": "CWE-672",
	"num": 1
},
{
	"id": "CWE-358",
	"num": 1
},
{
	"id": "CWE-388",
	"num": 1
},
{
	"id": "CWE-254",
	"num": 1
},
{
	"id": "CWE-310",
	"num": 1
}]
```